.changeset/smart-shoes-find.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cli": minor
+---
+
+comet generate-block-types generates now `AvailableBlockTypes` type, which contains a string union for all available block types
```

Comment:
I'd suggest changing the type to `AllBlocks` or `AvailableBlocks` since it represents the block names, not the block types. What do you think?

Reply:
sounds good. 👍

Reply:
updated

Reply:
> since it represents the block names

wouldn't be `AllBlockNames` be appropriate then?

Reply:
- `AllBlocks`
- `AllBlockNames`
- `AllBlockTypes`

would all be possible and valid namings in my opinion

Reply:
It should be either AllBlocks or AllBlockNames

Reply:
> but it's the type of the block and the function is also called generate-block-types

Yes, because it generates TypeScript types.

> but it's the type of the block

No really 😀